OutOfSchool/OutOfSchool.DataAccess/Models/Child.cs

```diff
@@ -9,18 +9,18 @@ public class Child
     {
         public long Id { get; set; }
 
-        [Required(ErrorMessage = "First name is required")]
-        [MaxLength(40)]
-        [RegularExpression(@"[\w\-\']*", ErrorMessage = "First name cannot contains digits and special symbols")]
+        [Required]
+        [MinLength(1)]
+        [MaxLength(30)]
```

Comment:
Shouldn't we have additional validation in data access if somehow someone manages create/modify child without going through child controller path (and childdto validation)? i.e., some other custom logic that will somehow modify child data.
p.s. question for discussion, not immediate action.